packages/plans-grid/src/plans-details/style.scss

```diff
@@ -1,6 +1,6 @@
-@import '../base-styles.scss';
-@import '../mixins.scss';
-@import '../variables.scss';
+@import '~@wordpress/base-styles/breakpoints';
+@import '~@wordpress/base-styles/mixins';
+@import '~@automattic/typography/styles/variables';
```

Comment:
This is done to import only the specific files that has the specific variables & mixins needed instead of blindly importing everything. You'll see this in a few other files as well.